src/main/java/bit/javaoop/CEO.java

```diff
@@ -0,0 +1,43 @@
+package bit.javaoop;
+
+
+public class CEO extends AbstractEmployee{
+    private final String position="ceo";
```

Comment:
I don't like this idea of tagging each employee with its position. It's against the idea of OOP and polymorphism. Let's look at simplified example. 

We have classes like `Animal`, `Cat`, `Dog` all of which have a method `giveVoice()`

The OOP way:
```
abstract class Animal {
   abstract String giveVoice();
}

class Dog extends Animal {
   String giveVoice() { 
       return "Woof woof!";
   }
}

class Cat extends Animal {
   String giveVoice() { 
       return "Meow meow!";
   }
}

...
// somewhere else

Animal animal = new Cat();
System.out.println(animal.giveVoice());

```

Your way (bad 😞):

```
abstract class Animal {
   abstract String animalType;
}

class Dog extends Animal {
     abstract String animalType = "dog";
}

class Cat extends Animal {
     abstract String animalType = "cat";
}

...
// somewhere else

Animal animal = new Cat();

if (animal.animalType == "cat") System.out.println("Meow meow!");
else System.out.println("Woof woof!");


```